test/OrchardCore.Commerce.Tests.UI/Tests/UserTests/UserPersistenceTests.cs

```diff
@@ -41,8 +41,9 @@ public UserPersistenceTests(ITestOutputHelper testOutputHelper)
         ExecuteTestAfterSetupAsync(
             async context =>
             {
-                await context.SignInDirectlyAsync();
-                await context.GoToRelativeUrlAsync("/user/addresses");
+                context.Missing(By.ClassName("user-addresses-widget"));
```

Comment:
Shouldn't we also check the other "widget" (`user-details-widget`) the same way? (I know it's technically one).

Here: https://github.com/OrchardCMS/OrchardCore.Commerce/blob/5f11e9b4e3e0da889d7c9f48ee999230b7227397/test/OrchardCore.Commerce.Tests.UI/Tests/UserTests/UserPersistenceTests.cs#L127-L128

By checking if the icon is missing if we are logged out, and clicking on it etc.

Reply:
I don't think there is a point to it, because it's the same template. Even checking the `context.Missing()` here felt kind of unnecessary, because we shouldn't have to test stock OC features such as the `Authenticated` layer working correctly.

Reply:
Alright then!